docker/mpp.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+apt-get -qq update
+apt install -y pkg-config
+
+if [ -e /usr/local/include/mpp/mpp.h ] || [ -e /usr/include/mpp/mpp.h ] || pkg-config --exists rockchip-mpp; then
+    HAS_MPP=1
+else
+# Install mpp from sources
+apt-get -qq update
+# TODO: move all apt-get installs to dedicated script for having it in a single Docker layer = performance of reruns.
+apt-get install -y git build-essential yasm pkg-config \
+libtool coreutils autoconf automake build-essential cmake \
+doxygen git graphviz imagemagick libasound2-dev libass-dev \
+libavcodec-dev libavdevice-dev libavfilter-dev libavformat-dev \
+libavutil-dev libfreetype6-dev libgmp-dev libmp3lame-dev \
+libopencore-amrnb-dev libopencore-amrwb-dev libopus-dev \
+librtmp-dev libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev \
+libsdl2-net-dev libsdl2-ttf-dev libsnappy-dev libsoxr-dev \
+libssh-dev libssl-dev libtool libv4l-dev libva-dev libvdpau-dev \
+libvo-amrwbenc-dev libvorbis-dev libwebp-dev libx264-dev libx265-dev \
+libxcb-shape0-dev libxcb-shm0-dev libxcb-xfixes0-dev libxcb1-dev \
+libxml2-dev lzma-dev meson nasm pkg-config python3-dev \
+python3-pip texinfo wget yasm zlib1g-dev libdrm-dev libaom-dev libdav1d-dev \
+libmp3lame-dev
+
+cd /tmp
+git clone https://github.com/rockchip-linux/mpp.git
+cd mpp
+mkdir build || true && cd build
+
+ARCH=$(uname -m)
+EXTRA_CFLAGS=""
+EXTRA_CXXFLAGS=""
+
+if [ "$ARCH" = "aarch64" ]; then
+    EXTRA_CFLAGS="-march=armv8-a+crc -mfpu=neon-fp-armv8 -mfloat-abi=hard"
+    EXTRA_CXXFLAGS="-march=armv8-a+crc -mfpu=neon-fp-armv8 -mfloat-abi=hard"
```

Comment:
I would get build errors with the `-mfpu=neon-fp-armv8 -mfloat-abi=hard` flags here and in the ffmpeg flags

Reply:
I'm testing this as maximum possible performance options and will be removing everything unstable

Reply:
The aarch64 gcc compiler does not have options for `mfpu` or `mfloat-abi`, so these will have to be arm32-only flags

Reply:
Just wanted to note, in case y'all didn't see the 0.12 RC release notes, future versions of frigate are not going to be supporting arm32 anymore so there is no need to worry about that 👍